flow2ml/Flow.py

```diff
@@ -246,4 +249,9 @@ def deployTensorflowModels(self,conversions,model):
       # Applying the convert function
       tflite_model = tf_lite_converter.convert()
       trained_models=TF_LITE_MODEL_FILE_NAME
-      open(trained_models,"wb").write(tflite_model)
\ No newline at end of file
+      open(trained_models,"wb").write(tflite_model)
+
+  def calculateImageQuality(self):
+    processed_data_folder = os.path.join(self.dataset_dir,'processedData')
+    self.image_scores = {}
+    self.create_scores_doc(processed_data_folder)
```

Comment:
Don’t create this doc in processedData folder. That would be difficult to user to find it if he have many images inside that folder.

Reply:
Okay. Where should I create it?

Reply:
Create folder “GeneratedReports/ImageQualityReport.docx, we will store all reports generated either related to preprocessing and postproocessing there.

Reply:
It would be better to create a new directory for storing main details regarding pre-processing, name it preProcessingOutcomes and store it in that directory.

Later we can add new functionalities in the future and store those outcomes there.

Reply:
Okay. @chebroluharika @yvkrishna should I create GeneratedReports directory inside the flow2ml folder and store it over there?